frontend/src/app/patients/AddPatient.tsx

```diff
@@ -140,6 +234,10 @@ const AddPatient = () => {
     return <Redirect to={redirect} />;
   }
 
+  if (goBack) {
+    history.goBack();
```

Comment:
is there a chance this would take you outside the app? like if you come directly to the add patient link from another website

Reply:
Indeed it could. The other option would be to redirect to the test queue like we do elsewhere. The `goBack()` is probably a little nicer when you _are_ coming from within SimpleReport since it won't break a user out of whatever workflow they were in, but as you point out it's less desirable if someone were to land on the `Add Person` page from outside the app.

Reply:
does re-routing to the `people page `makes sense,

- Most people will be coming in from there, so it behaves like a go back
- It will capture the weird case of people coming in from outside

Reply:
I'm fine with that. @nullflux and I consciously moved away from this approach initially but I think it was just mild preference.

Reply:
Addressed in 50e3731.